### DISCUSSION TITLE
How to load pytest plugins in a predefined order ?

### DISCUSSION THREAD

**Discussion by harmin-parra** [pytest-dev/pytest#11482] [Ideas]
I want to run pytest with 2 plugins: **pluginA** and **pluginB**

Both plugins implement the ``pytest_configure`` hook

I want **pluginA** to be loaded first, so that its ``pytest_configure`` function executes before than **pluginB**.

How can I achieve that ?

How can I defined precedence rules to pytest to load plugins ?

**Comment by RonnyPfannschmidt** [comment]
How are those Plugins loaded to begin with?

**Reply by harmin-parra** [reply]
The plugins enter into action if I set some command-line options when executing pytest.

The plugins have options configured via `pytest_addoptions`

I do know there are `trylast` and `tryfirst` parameters for the ``@pytest.hookimpl marker``

But if I use `tryfirst=True`, I have a crash, maybe because it executes before pytest-html configuration ???

**Reply by RonnyPfannschmidt** [reply]
there currently is no builtin order/priority system 

as no extra details on the plugins relation is provided i cant provide any more suggestions
